.travis.yml

```diff
@@ -13,4 +13,4 @@ jobs:
   include:
     - stage: tests
       script:
-      - make travis-runtests
+      - make publish-travis
```

Comment:
Update docs for this change from travis-runtests to publish-travis.

Reply:
Okay, I'll do it with other part (this is still not done btw)